### PR TITLE
feat(paymentMethodDescription) saved paymentMethodDescription into authorizationRequestEvent

### PR DESCRIPTION
**Pull request by Fernando-Granato**
Depends on https://github.com/pagopa/pagopa-ecommerce-commons/pull/102
Depends on https://github.com/pagopa/pagopa-infra/pull/1156

#### List of Changes
👉🏻 The change was made to save the value relating to the payment method description returned by the calculateFee during the authorization request.

#### Motivation and Context
The field has been saved to enhance the parameter in the payment confirmation email 🙂

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/test/java/it/pagopa/transactions/commands/handlers/TransactionRequestAuthorizizationHandlerTest.java [hunk 1]
```diff
@@ -155,6 +155,7 @@ void shouldSaveAuthorizationEvent() {
                 "brokerName",
                 "pspChannelCode",
                 "paymentMethodName",
+                "paymentMethodDescription",
```
Comment (pietro-tota) [inline src/test/java/it/pagopa/transactions/commands/handlers/TransactionRequestAuthorizizationHandlerTest.java:158]: add an assertion on queue written event to check that paymentMethodDescription is effectively written into queue event

**Review by pietro-tota — APPROVED** [review]
Well done, just a comment

File: src/test/java/it/pagopa/transactions/commands/handlers/TransactionRequestAuthorizizationHandlerTest.java (hunk 1 above)
Comment (Fernando-Granato) [reply src/test/java/it/pagopa/transactions/commands/handlers/TransactionRequestAuthorizizationHandlerTest.java:158]: Fixed with 👉🏻 232f60f1d7090c81fb2d1b03d23172e9a73d710d

**Review by pietro-tota — APPROVED** [review]
🚀

**Review by pietro-tota — APPROVED** [review]

**Pull request merged**